docs/diagrams/StudentUUID.puml

```diff
@@ -0,0 +1,10 @@
+@startuml
```

Comment:
I think we should include `style.puml` across all our diagrams to ensure consistency?